subprojects/packagefiles/box2d/meson.build

```diff
@@ -13,4 +13,7 @@ box2d_dep = declare_dependency(link_with: box2d_lib,
     include_directories: incdir,
     version: meson.project_version())
 
-subdir('unit-test')
+# Upstream has an old versionm of doctest.h which is imcompatible with glibc >=2.34
```

Comment:
Sounds like we should be using *our* doctest subproject instead.

Reply:
I....don't know how possible that is. There's a doctest.h in that directory.

Reply:
Speficially, the error is:

```
../subprojects/box2d-2.4.1/unit-test/doctest.h:4021:47: error: size of array ‘altStackMem’ is not an integral constant-expression
 4021 |         static char             altStackMem[4 * SIGSTKSZ];
```

Reply:
Huh. It's possible with implicit_include_directories: false . Cool.